packages/core/src/browser/style/tree.css

```diff
@@ -139,3 +139,51 @@
 .theia-tree-element-node {
     width: 100%
 }
+
+.theia-TreeNodeIndent {
```

Comment:
new class names should follow current guideliens: https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#css-use-lower-case-with-dashes